src/components/home_complaints/index.js

```diff
@@ -0,0 +1,181 @@
+import React from "react";
+import { connect } from "react-redux";
+import { Link } from "react-router-dom";
+
+import { Loading } from "formula_one";
+
+import locals from "./index.css";
+import styles from "core/index.css";
+
+import { complaintsUrl } from "../../urls";
+import { getComplaints } from "../../actions/complaints";
+import { addComplaint } from "../../actions/add_complaint";
+import moment from "moment";
+import { urlStaticBase } from "../../urls";
+
+class HomeComplaints extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      complaints: {},
+      max_length: 6,
```

Comment:
-> Define as a constant - the value is not going to change

Reply:
Removed. Unnecessary variable.